platform-emf/src/main/java/com/softicar/platform/emf/form/EmfFormBody.java

```diff
@@ -85,6 +85,7 @@ public void enterEditMode() {
 			if (ensureEntityIsFresh()) {
 				upperPart.enterEditMode();
 				lowerPart.showSectionContainer(new EmfFormSaveOrCancelActions<>(this, upperPart.getAttributesDiv()));
+				form.enterEditMode();
```

Comment:
these calls are rather notification calls, they are not expected to do anything by default, so I think should rather be named something with `notify`...and I think we should aggregate these methods into a single `nofityModeChange(EmfFormMode)` and then have an enum  `EmfFormMode { CREATE, EDIT, VIEW }` or something

Reply:
actually, why do we need these calls at all???